apps/timetable/ui/js/index.js

```diff
@@ -81,6 +93,15 @@ define(['gh.core', 'gh.constants', 'validator', 'gh.calendar', 'gh.header', 'gh.
      * @private
      */
     var addBinding = function() {
+
+        // Show the empty timetable page
+        $(document).on('gh.empty.timetable', function(evt, data) {
+
+            // Show the empty timetable
+            showEmptyTimetable(data);
```

Comment:
Let's rewrite this as `$(document).on('gh.empty.timetable', showEmptyTimetable);` and document the event parameter on the function